examples/openpsi/chat.scm

```diff
@@ -1,121 +1,85 @@
 ; Copyright (C) 2016 OpenCog Foundation
 ; This is a simple example of how an OpenPsi driven dialogue system may look like
 
-(use-modules (opencog)
-             (opencog nlp)
-             (opencog nlp relex2logic)
-             (opencog nlp fuzzy)
-             (opencog nlp chatbot)
-             (opencog exec)
-             (opencog openpsi))
-
-(load-r2l-rulebase)
-(set! relex-server-host "172.17.0.2")
```

Comment:
I haven't touched OpenPsi. And i did merge master just before creating this pull request not sure how this got here.